### DISCUSSION TITLE
wire:click in Tailwind CSS not working

### DISCUSSION THREAD

**Discussion by kevinwilliam7** [livewire/livewire#3622] [1. Help]
im trying install tailwind css to my project but i found a error where my wire:click is not working, there's a solution?
![image](https://user-images.githubusercontent.com/76833135/131254459-afd7f713-c568-4144-a82c-660d7e054435.png)
![image](https://user-images.githubusercontent.com/76833135/131254480-ff3ce530-c8e3-44a7-9f4f-f884010fc20b.png)

**Comment by joshhanley** [comment]
@lingelson7 can you confirm was it working before you added in tailwind?

Is there any errors in console in your browser dev tools?

Can you share your layout file too please?

Please don't paste screenshots, can you actually paste code?

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

Thanks!
